### ISSUE TITLE
[NuGet.org Bug]: Checkboxes have different "styles"

### ISSUE THREAD

**Issue by robertmuehsig** [NuGet/NuGetGallery#10407]
### Impact

It bothers me. A fix would be nice

### Describe the bug

Hi,

I noticed that some checkboxes uses the wrong style. I guess this is how the checkbox should be appear:

![Image](https://github.com/user-attachments/assets/f4c6635c-c21c-4c76-9d50-761d2bcb14a3)

Notice the "white"-checkmark and the focus around the checkbox + label.

Some checkboxes, like the "unlist" checkbox, are currently look like this:

![Image](https://github.com/user-attachments/assets/ef99c507-b9e9-4de7-afad-05c243c1ae80)

Notice the "black"-checkmark (which is hard to see) and the focus is only around the checkbox, but not the label.

The reason behind the issue is, that the structure is wrong on these checkboxes, e.g.:

```
                        <div class="row-checkbox-label">
                            @Html.ShowCheckboxFor(package => package.Listed, customClass: "brand-checkbox")
                            @Html.ShowLabelFor(package => package.Listed, "List in search results.", isrequired: false)
                        </div>
```

...should be...

```
                        <div class="row-checkbox-label">
                            <label class="brand-checkbox">
                                @Html.ShowCheckboxFor(package => package.Listed)
                                <span>List in search results.</span>
                            </label>
                        </div>
```

... which would result in this:

![Image](https://github.com/user-attachments/assets/80953f92-e72d-44a4-be88-9158d9d723b9)

I assume that all `ShowCheckboxFor...(customClass: "brand-checkbox")` are affected by this issue:

![Image](https://github.com/user-attachments/assets/11a391da-7e80-49a5-8c91-02048422c6c9)

### Repro Steps

Try to unlist some packages - the checkbox is not correct.

### Expected Behavior

"Readable" checkboxes.

### Additional Context and logs

I would be happy to fix this with a PR (if my analysis is correct and the proposed fix looks good to you).

**Comment by erdembayar** [comment]
@ryuyu 
Is this issue could our accessibility CSE address?

**Comment by erdembayar** [comment]
> @erdembayar I could send you a PR for this issue if you want. Just wanted to ask first if my analysis is correct.

Currently waiting for @ryuyu  to take a look, I'm not really familiar with this accessibility requirements.

**Comment by ryuyu** [comment]
This looks like a real issue. Great catch @robertmuehsig!
If you want to send a PR for this, please feel free and tag me directly and I'll pull it in.
If not, we will pick it up soon.

Really appreciate the report!
